bindings/rust/s2n-tls-sys/build.rs

```diff
@@ -205,16 +206,10 @@ fn builder() -> cc::Build {
 fn build_cmake() {
     let mut config = cmake::Config::new("lib");
 
-    // sometimes openssl-sys decides not to set this value so we may need to set it anyway
-    if option_env("DEP_OPENSSL_ROOT").is_none() {
-        let include = env("DEP_OPENSSL_INCLUDE");
-        if let Some(root) = Path::new(&include).parent() {
-            std::env::set_var("DEP_OPENSSL_ROOT", root);
-        }
-    }
+    let libcrypto = Libcrypto::default();
 
     config
-        .register_dep("openssl")
+        .register_dep(&format!("aws_lc_{}", libcrypto.version))
```

Comment:
nit: it feels like the `aws_lc` prefix information should be owned by the libcrypto struct.

Reply:
funny you mention that... that's actually what i had at first, but since this is conditionally enabled based on the `cmake` feature, i ended up with a dead_code warning and figured it was just cleaner to do it inline.